### PR TITLE
[BULK] - DocuTune remediation - Sensitive terms with GUIDs (part 14)

### PR DESCRIPTION
**Pull request by meganbradley**
Applying sensitive terms with GUID changes as part of Content SFI and outlined in [Overview - Writing content securely - Platform Manual](https://review.learn.microsoft.com/en-us/help/platform/security-reference?branch=main). Changes are part of the Microsoft-wide SFI effort. Point of contact: @CelesteDG

DocuTune v1.5.2.0
CorrelationId: [ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db](https://github.com/Azure/azure-docs-powershell-azuread/pulls?q=is%3Apr+ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db+)

#docutune

### CODE REVIEW

**Comment by meganbradley** [comment]
#docutune-review

File: azureadps-2.0/AzureAD/Get-AzureADGroupOwner.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The Get-AzureADGroupOwner cmdlet gets an owner of a group in the Microsoft Entra
 
 ### Example 1: Get a group owner by ID
 ```
-PS C:\>Get-AzureADGroupOwner -ObjectId "62438306-7c37-4638-a72d-0ee8d9217680"
+PS C:\>Get-AzureADGroupOwner -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADGroupOwner.md:26]: Object ID GUID in the output isn't updated.

File: azureadps-2.0/AzureAD/Get-AzureADGroupMember.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ examples:
   summary: ""
 - title: 'Example 2: Get all members within a group by group ID'
   code: |-
-    PS C:\> Get-AzureADGroupMember -ObjectId "12431118-5c12-6653-h82e-1ee8d9217682" -All $true
+    PS C:\> Get-AzureADGroupMember -ObjectId "bbbbbbbb-1111-2222-3333-cccccccccccc" -All $true
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADGroupMember.yml:20]: Object IDs in the output aren't updated.

File: azureadps-2.0/AzureAD/Get-AzureADGroupOwner.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Get a group owner by ID'
   code: |-
-    PS C:\>Get-AzureADGroupOwner -ObjectId "62438306-7c37-4638-a72d-0ee8d9217680"
+    PS C:\>Get-AzureADGroupOwner -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADGroupOwner.yml:11]: Object ID in the output isn't updated.

File: azureadps-2.0/AzureAD/Get-AzureADMSApplicationExtensionProperty.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ Retrieves the list of extension properties on an application object.
 
 ### Example 1: Get extension properties
 ```
-PS C:\>Get-AzureADMSApplicationExtensionProperty -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84"
+PS C:\>Get-AzureADMSApplicationExtensionProperty -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADMSApplicationExtensionProperty.md:26]: Object ID GUID not updated in the output.

File: azureadps-2.0/AzureAD/Get-AzureADMSApplicationOwner.md [hunk 5]
```diff
@@ -23,7 +23,7 @@ Retrieves the list of owners for an application object.
 
 ### Example 1: Get the owner of an application
 ```
-PS C:\>Get-AzureADMSApplicationOwner -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84" -Top 1
+PS C:\>Get-AzureADMSApplicationOwner -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -Top 1
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADMSApplicationOwner.md:26]: Object ID not updated in the output.

File: azureadps-2.0/AzureAD/Get-AzureADMSApplicationExtensionProperty.yml [hunk 6]
```diff
@@ -14,7 +14,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Get extension properties'
   code: |-
-    PS C:\>Get-AzureADMSApplicationExtensionProperty -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84"
+    PS C:\>Get-AzureADMSApplicationExtensionProperty -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADMSApplicationExtensionProperty.yml:17]: Object ID not updated in the output.

File: azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.md [hunk 7]
```diff
@@ -38,19 +38,19 @@ PS C:\> Get-AzureADMSRoleAssignment -Filter "roleDefinitionId eq '62e90394-69f5-
 
 RoleDefinitionId                     PrincipalId                          ResourceScope Id
 ----------------                     -----------                          ------------- --
-62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRqAVAnJQyBBLv1lM-1
+62e90394-69f5-4237-9190-012177145e34 aaaaaaaa-bbbb-cccc-1111-222222222222 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRqAVAnJQyBBLv1lM-1
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.md:41]: PrincipalId GUID not updated for all entries in the output.

File: azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.yml [hunk 8]
```diff
@@ -26,18 +26,18 @@ examples:
 
     RoleDefinitionId                     PrincipalId                          ResourceScope Id
     ----------------                     -----------                          ------------- --
-    62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRqAVAnJQyBBLv1lM-1
+    62e90394-69f5-4237-9190-012177145e34 aaaaaaaa-bbbb-cccc-1111-222222222222 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRqAVAnJQyBBLv1lM-1
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.yml:29]: Not all PrincipalID GUIDs have been updated in the output.

File: azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.yml [hunk 9]
```diff
@@ -26,18 +26,18 @@ examples:
...
   code: |-
-    PS C:\> Get-AzureADMSRoleAssignment -Filter "principalId eq '69584002-b4d1-4055-9c94-320542efd653'"
+    PS C:\> Get-AzureADMSRoleAssignment -Filter "principalId eq 'aaaaaaaa-bbbb-cccc-1111-222222222222'"
 
     RoleDefinitionId                     PrincipalId                          ResourceScope Id
     ----------------                     -----------                          ------------- --
-    89c55b63-78c3-478b-b79e-074d0e87269e 69584002-b4d1-4055-9c94-320542efd653 /             Y1vFBcN4i0e3ngdNDocmngJAWGnRtGDAnJQyBBLv1lM-1
-    62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRtGDAnJQyBBLv1lM-1
+    89c55b63-78c3-478b-b79e-074d0e87269e aaaaaaaa-bbbb-cccc-1111-222222222222 /             Y1vFBcN4i0e3ngdNDocmngJAWGnRtGDAnJQyBBLv1lM-1
+    62e90394-69f5-4237-9190-012177145e34 aaaaaaaa-bbbb-cccc-1111-222222222222 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRtGDAnJQyBBLv1lM-1
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.yml:40]: Same here.

File: azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.md [hunk 10]
```diff
@@ -24,14 +24,14 @@ Retrieves the object(s) specified by the objectIds parameter
 
 ### Example 1
 ```
-PS C:\WINDOWS\system32> Get-AzureADObjectByObjectId -ObjectIds 2af3478a-27da-4837-a387-b22b3fb236a8, c4fdf87f-f68e-4859-8bcf-36579b66005e
+PS C:\WINDOWS\system32> Get-AzureADObjectByObjectId -ObjectIds aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb, c4fdf87f-f68e-4859-8bcf-36579b66005e
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.md:27]: Not all Object Id values have been updated in the command and in the output as well.

File: azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.yml [hunk 11]
```diff
@@ -17,14 +17,14 @@ syntaxes:
 examples:
 - title: Example 1
   code: |-
-    PS C:\WINDOWS\system32> Get-AzureADObjectByObjectId -ObjectIds 2af3478a-27da-4837-a387-b22b3fb236a8, c4fdf87f-f68e-4859-8bcf-36579b66005e
+    PS C:\WINDOWS\system32> Get-AzureADObjectByObjectId -ObjectIds aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb, c4fdf87f-f68e-4859-8bcf-36579b66005e
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.yml:20]: Same as previous comment.

File: azureadps-2.0/AzureAD/Get-AzureADUserCreatedObject.md [hunk 12]
```diff
@@ -23,7 +23,7 @@ The Get-AzureADUserCreatedObject cmdlet gets objects created by a user in Azure
 
 ### Example 1: Get a user-created object
 ```
-PS C:\>Get-AzureADUserCreatedObject -ObjectId "df19e8e6-2ad7-453e-87f5-037f6529ae16"
+PS C:\>Get-AzureADUserCreatedObject -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADUserCreatedObject.md:26]: All GUIDs in the output have not been updated.

File: azureadps-2.0/AzureAD/Get-AzureADMSApplicationOwner.yml [hunk 13]
```diff
@@ -28,7 +28,7 @@ examples:
   summary: ""
 - title: 'Example 1: Get the owners of an application'
   code: |-
-    PS C:\>Get-AzureADMSApplicationOwner -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84" -All $true
+    PS C:\>Get-AzureADMSApplicationOwner -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -All $true
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Get-AzureADMSApplicationOwner.yml:31]: GUIDs in the output haven't been updated.

**Review by msewaweru — CHANGES_REQUESTED** [review]
For most files, the GUIDs in the output aren't updated.

File: azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.md [hunk 14]
```diff
@@ -38,19 +38,19 @@ PS C:\> Get-AzureADMSRoleAssignment -Filter "roleDefinitionId eq '62e90394-69f5-
...
 62e90394-69f5-4237-9190-012177145e34 3f5e48d2-6bf4-4bf3-a4ff-8bbd9f23ed0b /             lAPpYvVpN0KRkAEhdxReENJPEj_0a_NLpP-LvZ8D7Qs-1
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.md:42]:
```suggestion
62e90394-69f5-4237-9190-012177145e34 aaaaaaaa-bbbb-cccc-1111-222222222222 /             lAPpYvVpN0KRkAEhdxReENJPEj_0a_NLpP-LvZ8D7Qs-1
```

File: azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.yml [hunk 15]
```diff
@@ -26,18 +26,18 @@ examples:
 
     RoleDefinitionId                     PrincipalId                          ResourceScope Id
     ----------------                     -----------                          ------------- --
-    62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRqAVAnJQyBBLv1lM-1
+    62e90394-69f5-4237-9190-012177145e34 aaaaaaaa-bbbb-cccc-1111-222222222222 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRqAVAnJQyBBLv1lM-1
     62e90394-69f5-4237-9190-012177145e34 3f5e48d2-6bf4-4bf3-a4ff-8bbd9f23ed0b /             lAPpYvVpN0KRkAEhdxReENJPEj_0a_NLpP-LvZ8D7Qs-1
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.yml:30]:
```suggestion
    62e90394-69f5-4237-9190-012177145e34 aaaaaaaa-bbbb-cccc-1111-222222222222 /             lAPpYvVpN0KRkAEhdxReENJPEj_0a_NLpP-LvZ8D7Qs-1
```

File: azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.yml [hunk 16]
```diff
@@ -26,18 +26,18 @@ examples:
...
-    PS C:\> Get-AzureADMSRoleAssignment -Filter "principalId eq '69584002-b4d1-4055-9c94-320542efd653'"
+    PS C:\> Get-AzureADMSRoleAssignment -Filter "principalId eq 'aaaaaaaa-bbbb-cccc-1111-222222222222'"
 
     RoleDefinitionId                     PrincipalId                          ResourceScope Id
     ----------------                     -----------                          ------------- --
-    89c55b63-78c3-478b-b79e-074d0e87269e 69584002-b4d1-4055-9c94-320542efd653 /             Y1vFBcN4i0e3ngdNDocmngJAWGnRtGDAnJQyBBLv1lM-1
-    62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRtGDAnJQyBBLv1lM-1
+    89c55b63-78c3-478b-b79e-074d0e87269e aaaaaaaa-bbbb-cccc-1111-222222222222 /             Y1vFBcN4i0e3ngdNDocmngJAWGnRtGDAnJQyBBLv1lM-1
+    62e90394-69f5-4237-9190-012177145e34 aaaaaaaa-bbbb-cccc-1111-222222222222 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRtGDAnJQyBBLv1lM-1
     eb1d8c90-acf5-460d-8424-c1f1a6fbdb85 69584002-b4d1-4055-9c94-320542efd653 /             NIwd6_WsDUaEJMHxpvvbhQJAWGnRtGDAnJQyBBLv1lM-1
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Get-AzureADMSRoleAssignment.yml:41]:
```suggestion
    eb1d8c90-acf5-460d-8424-c1f1a6fbdb85 aaaaaaaa-bbbb-cccc-1111-222222222222 /             NIwd6_WsDUaEJMHxpvvbhQJAWGnRtGDAnJQyBBLv1lM-1
```

File: azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.md (hunk 10 above)
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.md:27]:
```suggestion
PS C:\WINDOWS\system32> Get-AzureADObjectByObjectId -ObjectIds aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb, bbbbbbbb-1111-2222-3333-cccccccccccc
```

File: azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.md [hunk 17]
```diff
@@ -24,14 +24,14 @@ Retrieves the object(s) specified by the objectIds parameter
...
 
 ObjectId                             AppId                                DisplayName
 --------                             -----                                -----------
 c4fdf87f-f68e-4859-8bcf-36579b66005e 71715b24-8cdd-432b-a138-86e8ad179274 Woodgrove HR App
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.md:31]:
```suggestion
bbbbbbbb-1111-2222-3333-cccccccccccc 71715b24-8cdd-432b-a138-86e8ad179274 Woodgrove HR App
```

File: azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.yml (hunk 11 above)
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.yml:20]:
```suggestion
    PS C:\WINDOWS\system32> Get-AzureADObjectByObjectId -ObjectIds aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb, bbbbbbbb-1111-2222-3333-cccccccccccc
```

File: azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.yml [hunk 18]
```diff
@@ -17,14 +17,14 @@ syntaxes:
...
 
     ObjectId                             AppId                                DisplayName
     --------                             -----                                -----------
     c4fdf87f-f68e-4859-8bcf-36579b66005e 71715b24-8cdd-432b-a138-86e8ad179274 Woodgrove HR App
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.yml:24]:
```suggestion
    bbbbbbbb-1111-2222-3333-cccccccccccc 71715b24-8cdd-432b-a138-86e8ad179274 Woodgrove HR App
```

File: azureadps-2.0/AzureAD/Get-AzureADGroupMember.yml (hunk 2 above)
Comment (cilwerner) [reply azureadps-2.0/AzureAD/Get-AzureADGroupMember.yml:20]: Changes implemented here are ok for the purposes of this PR

File: azureadps-2.0/AzureAD/Get-AzureADGroupOwner.yml (hunk 3 above)
Comment (cilwerner) [reply azureadps-2.0/AzureAD/Get-AzureADGroupOwner.yml:11]: Changes implemented here are ok for the purposes of this PR

File: azureadps-2.0/AzureAD/Get-AzureADMSApplicationExtensionProperty.md (hunk 4 above)
Comment (cilwerner) [reply azureadps-2.0/AzureAD/Get-AzureADMSApplicationExtensionProperty.md:26]: Changes implemented here are ok for the purposes of this PR

File: azureadps-2.0/AzureAD/Get-AzureADMSApplicationExtensionProperty.yml (hunk 6 above)
Comment (cilwerner) [reply azureadps-2.0/AzureAD/Get-AzureADMSApplicationExtensionProperty.yml:17]: Changes implemented here are ok for the purposes of this PR

File: azureadps-2.0/AzureAD/Get-AzureADMSApplicationOwner.md (hunk 5 above)
Comment (cilwerner) [reply azureadps-2.0/AzureAD/Get-AzureADMSApplicationOwner.md:26]: Changes implemented here are ok for the purposes of this PR

File: azureadps-2.0/AzureAD/Get-AzureADMSApplicationOwner.yml (hunk 13 above)
Comment (cilwerner) [reply azureadps-2.0/AzureAD/Get-AzureADMSApplicationOwner.yml:31]: Changes implemented here are ok for the purposes of this PR

File: azureadps-2.0/AzureAD/Get-AzureADMSPermissionGrantConditionSet.yml [hunk 19]
```diff
@@ -32,7 +32,7 @@ examples:
   summary: ""
 - title: 'Example 3: Get a permission grant condition set'
   code: |-
-    PS C:\>Get-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
+    PS C:\>Get-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Get-AzureADMSPermissionGrantConditionSet.yml:35]:
```suggestion
    PS C:\>Get-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "6b5e999b-0ba8-4186-a106-e0296c1c4358"
```

File: azureadps-2.0/AzureAD/Get-AzureADUserCreatedObject.md (hunk 12 above)
Comment (cilwerner) [reply azureadps-2.0/AzureAD/Get-AzureADUserCreatedObject.md:26]: Ok for now.

File: azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.yml (hunk 11 above)
Comment (cilwerner) [reply azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.yml:20]: Ok for now.

File: azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.md (hunk 10 above)
Comment (cilwerner) [reply azureadps-2.0/AzureAD/Get-AzureADObjectByObjectId.md:27]: Ok for now.

**Pull request merged**